nservicebus/azure-service-bus/transaction-support.md

```diff
@@ -7,37 +7,66 @@ tags:
+* Finally, it does not allow send operation on different messaging entities in the same transaction. So for example a send to Queue1 and a send to Queue2 in the same scope, will throw an exception. Again this problem is negated using separate instances of `IEnlistmentNotification` on each send operation. 
+
+The benefit of this approach is that user code can use database connections and send messages to multiple endpoints while maintaining the aspects expected from a transaction. However the downside is that each operation is individual. Even though they are executed at the same moment in a transaction, it can happen in theory that one of the operations fails, while the others do not. This problem is again mitigated by using multiple layers of retry behavior to reduce the chance, so a short interruption in connectivity or broker outage will not have a significant impact and operations will eventually succeed. However a full outage right after the database transaction committed could lead to a send operation that did not execute. 
+
+The maximum guarantee that the transport can deliver in these versions is in effect `ReceiveOnly`. 
+
+Enabling/Disabling DTC will have no effect, this setting is ignored either way as Azure Service Bus does not support it.
+
+Disabling transactions will also turn off the `PeekLock` mechanism, so that the transport immediately completes any incoming message before processing and shows no retry behavior. 
+  
+
+### Versions 7 and above
+
+In NServiceBus version 6 and above, the shape of the pipeline has changed so that a transaction scope is no longer mandatory in order to orchestrate dispatching of messages with complete/rollback behavior. The new architecture is schematically represented like this:
+
+[TODO: insert picture 2]
+
+Note the fork in the pipeline which is separating the user code invocation path from the dispatching path. By putting a suppress scope around this section of the pipeline, the transport can prevent any other transactional resource from enlisting in the scope.
+
+Furthermore the Azure Service Bus transport now also takes advantage of a little known capability if the Azure Service Bus SDK, the via entity path / transfer queue. Using this feature, send operations to different Azure Service Bus entities can be executed via a single entity, usually the receive queue. Schematically it works like this:
+
+[TODO: insert picture 3]
+
+Combining these capabilities, allows Azure Service Bus Transport to support `SendAtomicWithReceive`, `ReceiveOnly` and `None` transaction mode levels.
```

Comment:
@yvesgoeleven I tend to agree with @danielmarbach about adding the note here. When using ASB transport and creating satellites, it would be relevant to find it here first, and a note in the satellite doco. Perhaps a shared include file?